bundles/EcommerceFrameworkBundle/Controller/PricingController.php

```diff
@@ -46,7 +46,7 @@ public function onKernelController(FilterControllerEvent $event)
     }
 
     /**
-     * @Route("/list", methods={"GET"})
+     * @Route("/list", name="pimcore_ecommerceframework_pricing_list", methods={"GET"})
```

Comment:
please update js:
https://github.com/pimcore/pimcore/blob/c62924ca5c3d0c482a71097cb0231aff143208d6/bundles/EcommerceFrameworkBundle/Resources/public/js/pricing/config/panel.js#L160
and others too:
https://github.com/pimcore/pimcore/blob/c62924ca5c3d0c482a71097cb0231aff143208d6/bundles/EcommerceFrameworkBundle/Resources/public/js/pricing/config/panel.js#L50
https://github.com/pimcore/pimcore/blob/c62924ca5c3d0c482a71097cb0231aff143208d6/bundles/EcommerceFrameworkBundle/Resources/public/js/pricing/config/panel.js#L142
https://github.com/pimcore/pimcore/blob/c62924ca5c3d0c482a71097cb0231aff143208d6/bundles/EcommerceFrameworkBundle/Resources/public/js/pricing/config/panel.js#L249
https://github.com/pimcore/pimcore/blob/c62924ca5c3d0c482a71097cb0231aff143208d6/bundles/EcommerceFrameworkBundle/Resources/public/js/pricing/config/panel.js#L287
https://github.com/pimcore/pimcore/blob/c62924ca5c3d0c482a71097cb0231aff143208d6/bundles/EcommerceFrameworkBundle/Resources/public/js/pricing/config/panel.js#L316

Reply:
done